repo/packages/R/riak/0/marathon.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "{{riak.framework-name}}",
+  "instances": {{riak.instances}},
+  "cpus": {{riak.cpus}},
+  "mem": {{riak.mem}},
+  "ports": [0, 0],
+  "uris": ["{{riak.url}}"],
+  "env": {},
+  "args": [
```

Comment:
When testing locally for whatever reason, I noticed that I could either supply an args block OR a cmd block, and it was easier on the eyes to have everything in the args block. I just committed a change to have all of the args in the cmd block.